### PR TITLE
'OpenShift: New Service' command to command palette

### LINKED ISSUE #318: Add 'OpenShift: New Service' command to command palette

### PR DESCRIPTION
**Pull request by sudhirverma**
Fix #318.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -204,6 +204,11 @@
         "title": "New Service",
         "category": "OpenShift"
       },
+      {
```
Comment (dgolovin) [inline package.json]: No need for new command, because we can use the same one everywhere.

**Comment by sudhirverma** [comment]
@dgolovin please review this pr

**Comment by sudhirverma** [comment]
@dgolovin please review this pr

File: src/openshift/service.ts [hunk 2]
```diff
@@ -8,38 +8,79 @@ import { OpenShiftObject, Command } from '../odo';
...
-                Command.createService(project.getName(), application.getName(), serviceTemplateName, serviceTemplatePlanName, serviceName.trim()))
-                .then(() => Service.explorer.refresh(application))
-                .then(() => `Service '${serviceName}' successfully created`)
-                .catch((err) => Promise.reject(`Failed to create service with error '${err}'`));
+            return null;
         }
-        return null;
+    }
+
+    static async getApplicationData(application) {
```
Comment (jrichter1) [inline src/openshift/service.ts]: Please see https://github.com/redhat-developer/vscode-openshift-tools/pull/432#discussion_r245595571 and #499 for the summary of my issues with this implementation.

**Pull request merged**